training-net/src/Models/Comment.cs

```diff
@@ -0,0 +1,19 @@
+using System;
```

Comment:
be consistent. If you put an empty line between properties, keep the same criteria. when we use DataAnnotations, we put a line between properties anyway